test/powershell/Modules/Microsoft.PowerShell.Utility/Enable-RunspaceDebug.Tests.ps1

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+$FeatureEnabled = $EnabledExperimentalFeatures.Contains('Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints')
+
+Describe "`Enable-RunspaceDebug -Breakpoint` Unit Tests - Feature-Enabled" -Tags "CI" {
+
+    BeforeAll {
+        if (!$FeatureEnabled) {
+            Write-Verbose "Test Suite Skipped. The test suite requires the experimental feature 'Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints' to be enabled." -Verbose
+            $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+            $PSDefaultParameterValues["it:skip"] = $true
+            return
+        }
+
+        #Set up script file 1
+        $scriptFileName1 = Join-Path $TestDrive -ChildPath breakpointTestScript.ps1
+
+        $contents = @"
+function Hello
+{
+    `$greeting = 'Hello, world!'
+    write-host `$greeting
+}
+
+function Goodbye
+{
+    `$message = 'Good bye, cruel world!'
+    write-host `$message
+}
+
+Hello
+Goodbye
+"@
+
+        $contents > $scriptFileName1
+
+        # Set up script file 2
+        $scriptFileName2 = Join-Path -Path $PSScriptRoot -ChildPath psbreakpointtestscript.ps1
+
+        "`$var = 1 " > $scriptFileName2
+
+        $iss = [initialsessionstate]::CreateDefault2();
+        $testRunspace1 = [runspacefactory]::CreateRunspace($iss)
+        $testRunspace1.Name = "TestRunspaceDebuggerReset"
+        $testRunspace1.Open()
+    }
+
+    AfterAll {
+        if (!$FeatureEnabled) {
+            $global:PSDefaultParameterValues = $originalDefaultParameterValues
+            return
+        }
+
+        # clean up
+        Remove-Item -Path $scriptFileName1 -Force -ErrorAction SilentlyContinue
```

Comment:
We don't need to remove test script files if they are created under $TestDrive path.

Reply:
I see this was fixed in another commit.